packages/manager/modules/pci/src/projects/project/training/jobs/info/info.controller.js

```diff
@@ -75,21 +75,21 @@ export default class PciTrainingJobsInfoController {
       duration += moment().diff(this.start) / 1000;
     }
 
+    // eslint-disable-next-line no-underscore-dangle
+    const date = moment(moment.duration(duration, 'seconds')._data);
```

Comment:
Looks weird that we need to access `_data` from a moment object (also the naming looks like it is meant to be an internal property)
What is the purpose here ? `moment` probably expose some methods to allow to do this another way

Reply:
The main issue is that `moment.duration` does not have the same format abilities than moment and moment does not seem to expose a constructor from a duration. However this breaks with earlier versions of moment.
Instead I added a dedicated plugin: https://github.com/jsmreese/moment-duration-format